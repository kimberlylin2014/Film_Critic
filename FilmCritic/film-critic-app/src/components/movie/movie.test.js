import React from 'react';
import { shallow } from 'enzyme';
import Movie from './movie.component'

it('Test if Movie component is rendering', () => {
    const mockMovieProp = {Title: 'Taxi', Poster: "https://m.media-amazon.com/images/M/MV5BMGVmMWNiMDktYjQ0Mi00MWIxLTk0N2UtN2ZlYTdkN2IzNDNlXkEyXkFqcGdeQXVyODE5NzE3OTE@._V1_SX300.jpg", Plot: "Short story."}
    const wrapper = shallow(<Movie {...mockMovieProp}/>)
    expect(wrapper).toMatchSnapshot()
})