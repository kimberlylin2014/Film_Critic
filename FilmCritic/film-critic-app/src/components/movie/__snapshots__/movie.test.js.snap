// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if Movie component is rendering 1`] = `
<div
  className="Movie"
>
  <div
    className="img"
  >
    <img
      alt="Taxi"
      height="250px"
      src="https://m.media-amazon.com/images/M/MV5BMGVmMWNiMDktYjQ0Mi00MWIxLTk0N2UtN2ZlYTdkN2IzNDNlXkEyXkFqcGdeQXVyODE5NzE3OTE@._V1_SX300.jpg"
      width="200px"
    />
  </div>
  <div
    className="details"
  >
    <h4>
      Taxi
    </h4>
    <p>
       
      Short story.
    </p>
    <div
      className="public-route text-right"
    >
      <withRouter(AccessPrivatePageModal)
        Plot="Short story."
        Poster="https://m.media-amazon.com/images/M/MV5BMGVmMWNiMDktYjQ0Mi00MWIxLTk0N2UtN2ZlYTdkN2IzNDNlXkEyXkFqcGdeQXVyODE5NzE3OTE@._V1_SX300.jpg"
        Title="Taxi"
        label="View Reviews and More!"
      />
    </div>
  </div>
</div>
`;
